index.html

```diff
@@ -73,6 +76,198 @@ <h3 class="social_title blue">LET’S CONNECT</h3>
+            <button type="button" class="default_btn">See More</button>
+          </div>
+        </div>
+
+        <!-- work card 3 -->
+        <div class="work_card">
+          <div class="work_preview">
+            <img src="./images/Snapshoot Portfolio (2).png" alt="Project 3" />
+          </div>
+          <div class="work_details">
+            <h2 class="work_title">Tonic</h2>
+            <ul class="work_info">
+              <li class="w_info_item">CANOPY</li>
+              <li class="w_info_item">Back End Dev</li>
+              <li class="w_info_item">2015</li>
+            </ul>
+            <p class="work_details_content">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="work_cats">
+              <li>Html</li>
+              <li>Css</li>
+              <li>Javascript</li>
+            </ul>
+            <button type="button" class="default_btn">See More</button>
+          </div>
+        </div>
+
+        <!-- work card 4-->
+        <div class="work_card">
+          <div class="work_preview">
+            <img src="./images/Snapshoot Portfolio (3).png" alt="Project 4" />
+          </div>
+          <div class="work_details">
+            <h2 class="work_title">Multi-Post Stories</h2>
+            <ul class="work_info">
+              <li class="w_info_item">CANOPY</li>
+              <li class="w_info_item">Back End Dev</li>
+              <li class="w_info_item">2015</li>
+            </ul>
+            <p class="work_details_content">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="work_cats">
+              <li>Html</li>
+              <li>Css</li>
+              <li>Javascript</li>
+            </ul>
+            <button type="button" class="default_btn">See More</button>
+          </div>
+        </div>
+      </section>
+      <!-- end of work section-->
+
+      <section class="about">
+        <h2 class="headline_title">About Myself</h2>
+        <p class="about_details">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <h3 class="social_title blue">LET’S CONNECT</h3>
+        <ul class="social_links">
+          <li class="social_link">
+            <a href="#"
+              ><img
+                class="social_icon"
+                src="./images/twitter.svg"
+                alt="Twitter"
+            /></a>
+          </li>
+          <li class="social_link">
+            <a href="#"
+              ><img
+                class="social_icon"
+                src="./images/Linkedin icon.svg"
+                alt="Linkedin"
+            /></a>
+          </li>
```

Comment:
Kindly ensure that the social icons are horizontally aligned as shown by the design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/198841571-77c77f01-22c7-4bf7-8307-486760450cc7.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/198841577-676b8dfc-96f9-472a-9eae-9d7500b14bf7.png)